### PR TITLE
Feat: render color for outputs of vela up command

### LINKED ISSUE #3903: [Feature] render color for outputs of vela up command
**Is your feature request related to a problem? Please describe.**

The command prints plain text:

```
$ vela up -f app.yaml
Applying an application in vela K8s object format...
I0516 15:45:18.123356   27156 apply.go:107] "creating object" name="first-vela-app" resource="core.oam.dev/v1beta1, Kind=Application"
✅ App has been deployed 🚀🚀🚀
    Port forward: vela port-forward first-vela-app
             SSH: vela exec first-vela-app
         Logging: vela logs first-vela-app
      App status: vela status first-vela-app
        Endpoint: vela status first-vela-app --endpoint
Application prod/first-vela-app applied.
```

We need some colorful prints for the important message.

**Describe the solution you'd like**

Add some color for the prints.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by JarHMJ**
Signed-off-by: huangminjie <minjie.huang@daocloud.io>

render color for outputs of vela up command (#3903 )

### Description of your changes

Fixes #3903 

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [x] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

**Comment by JarHMJ** [comment]
image

**Comment by wonderflow** [comment]
image

I  hope these commands can have color

**Comment by JarHMJ** [comment]
image
Is this style OK？

File: references/common/application.go [hunk 1]
```diff
@@ -465,13 +465,14 @@ func (o *AppfileOptions) apply(app *corev1beta1.Application, scopes []oam.Object
...
 func Info(app *corev1beta1.Application) string {
+	green := color.New(color.FgYellow)
 	appName := app.Name
 	var appUpMessage = "✅ App has been deployed 🚀🚀🚀\n" +
-		fmt.Sprintf("    Port forward: vela port-forward %s\n", appName) +
-		fmt.Sprintf("             SSH: vela exec %s\n", appName) +
-		fmt.Sprintf("         Logging: vela logs %s\n", appName) +
-		fmt.Sprintf("      App status: vela status %s\n", appName) +
-		fmt.Sprintf("        Endpoint: vela status %s --endpoint\n", appName)
+		"    Port forward: " + green.Sprintf("port-forward %s\n", appName) +
```
Comment (charlie0129) [inline references/common/application.go:471]: `vela` is missing here

**Comment by wonderflow** [comment]
For the last line, i hope only application name has color

**Comment by JarHMJ** [comment]
> For the last line, i hope only application name has color

image

**Review by zzxwill — APPROVED** [review]
Cool enhancement!

**Pull request merged**